src/styles/components/milestones.scss

```diff
@@ -0,0 +1,110 @@
+.milestones-wrapper {
+  overflow: auto;
+
+  &::-webkit-scrollbar {
+    width: 0px;
+    background: transparent;
+  }
+
+  .milestones {
+    display: inline-block;
+    position: relative;
+    padding: 24px 0;
+    list-style: none;
+
+    &:before {
+      position: absolute;
+      content: " ";
+      height: 3px;
+      top: auto;
+      bottom: 30px;
+      left: 56px;
+      right: 0;
+      width: 100%;
+      margin-bottom: 20px;
+      margin-left: 8px;
+      background-color: #c4c4c4;
+
+      &:last-child {
+        opacity: 0;
+      }
+    }
+
+    .milestones-item {
+      display: table-cell;
+      position: relative;
+      float: none !important;
+      width: 20%;
+      margin: 0 auto;
+      padding: 0 20px 0 0;
+      vertical-align: bottom;
+
+      &.current-milestone:after {
+        content: '';
+        position: absolute;
+        left: 60px;
+        top: 110px;
+        width: 0;
+        height: 0;
+        border-left: 16px solid transparent;
+        border-right: 16px solid transparent;
+        border-top: 16px solid $aurora-blue-dk;
+        clear: both;
+      }
+
+      .milestones-badge {
+        top: auto;
+        bottom: 14px;
+        left: 43px;
+        width: 26px;
+        height: 26px;
+        position: absolute;
+        margin-left: 20px;
+        z-index: 100;
+        border-radius: 50%;
+        background-color: #4f5d7c;
```

Comment:
this could be a variable in _color

Reply:
Color added to `_colors.scss` as new variable.